client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx

```diff
@@ -419,6 +419,9 @@ export const ApplicationsTableAnalyze: React.FC = () => {
     importWriteAccess = checkAccess(userScopes, importsWriteScopes),
     applicationWriteAccess = checkAccess(userScopes, applicationsWriteScopes);
 
+  const areAppsInWaves = () =>
```

Comment:
This doesn't need to be a function, it can just be a boolean variable. It is immediately called twice in the same render when it only needs to run the `some` loop once